main/reference/vstorage-ref.md

```diff
@@ -1,13 +1,128 @@
 # VStorage Reference
 
-See also:
+In the Agoric platform, VStorage is a key-value store that:
+
+- provides a **read-only interface where clients of the consensus layer** can query the stored data using specific paths.
```

Comment:
maybe want to clarify here, "read-only" to external, but contracts can write to it. something like this, so they don't think vstorage is read-only in general.